Ported/combat-bees/Assets/Scripts/Systems/BeeSystem.cs

```diff
@@ -93,7 +126,6 @@ public void Execute()
                     ECB.SetComponent(BeeEntities[i], new BeeProperties
                     {
                         BeeMode = BeeMode.FindResource,
-                        Aggressivity = BeeProperties[i].Aggressivity,
```

Comment:
@LauWerUnity : is removing this here a mistake?

Reply:
Well I was thinking SetComponent only updates the fields you specify and ignores default values, that's probably wrong though, will re-add all the unchanged field copies and explicitly set them to past value, safer.